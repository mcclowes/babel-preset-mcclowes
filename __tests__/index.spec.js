var config = require('../index')

describe('babel config', () => {
  it('exports correctly', () => {
    expect(config.presets).toMatchSnapshot()
    expect(config.plugins).toMatchSnapshot()
  })
})