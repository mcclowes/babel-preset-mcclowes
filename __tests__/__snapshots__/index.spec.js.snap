// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`babel config exports correctly 1`] = `
Array [
  Array [
    "env",
    Object {
      "modules": "commonjs",
      "targets": Object {
        "browsers": "> 1%",
        "node": "current",
      },
    },
  ],
  "react",
]
`;

exports[`babel config exports correctly 2`] = `
Array [
  "@babel/transform-class-properties",
  "@babel/transform-decorators-legacy",
  "@babel/plugin-proposal-class-properties",
  "@babel/plugin-proposal-decorators",
  "@babel/plugin-proposal-json-strings",
  "@babel/plugin-syntax-dynamic-import",
  "@babel/plugin-syntax-import-meta",
  "@babel/plugin-proposal-optional-chaining",
  "@babel/plugin-proposal-pipeline-operator",
  Array [
    "@babel/ramda",
    Object {
      "useES": false,
    },
  ],
  Array [
    "@babel/plugin-transform-runtime",
    Object {
      "polyfill": false,
      "regenerator": true,
    },
  ],
]
`;
